packages/components/src/button/src/Button.tsx

```diff
@@ -142,11 +148,11 @@ export function InnerButton(props: InnerButtonProps) {
         },
         "end-icon": {
             className: "o-ui-button-end-icon",
-            size: condensed ? sizeValue : embeddedIconSize(sizeValue)
+            size: condensed ? sizeValue : iconSize(sizeValue)
```

Comment:
This changes is a smell to me. The embeddedIconSize utility is to ensure all our components use the same sizes for embedded icons.

Does the button component is not standard anymore? Does the embeddedIconSize needs an update becomes it doesn't reflect our DA anymore?

Those questions should be answered before making the change.

Reply:
> Horizontal tile is still broken:

as for the image height some discussions have started here: https://github.com/gsoft-inc/sg-orbit/issues/898#issuecomment-1042368002

I was planning to at least fix the overflow then tackle the problem of the image not stretching as it's a different issue to me.

A new image will need to be produced too in order to be able to cover the whole image area without stretching the image, I don't like it anyways :D

Reply:
Got it

Reply:
> embeddedIconSize utility is to ensure all our components use the same sizes for embedded icons.

in that case a review of all the components shall be done, the main goal here is to make sure we don't go lower than 14px in font-size as 12px is hard to read. By doing this it was logical to bump the icon sizes in small components. This might be an error but I think we should then bump all icons sizes to avoid icons that are hard to decipher.

Reply:
Indeed added this to the Sync discussion as it can be not the path we want to take, take Lozenge as an example.